src/pudl/metadata/fields.py

```diff
@@ -2344,6 +2393,10 @@
         "type": "string",
         "description": "The code of the plant's ISO or RTO. NA if not reported in that year.",
     },
+    "key": {
+        "type": "string",
+        "description": "Key within block.",
+    },
```

Comment:
Without more context it's not clear what this means, and `key` is a very short and generic column name.

Reply:
Same as above, I've changed the naming for this table to hopefully make it more clear.